kubernetes/base/config/kube_config.py

```diff
@@ -420,7 +420,7 @@ def _load_oid_token(self, provider):
         return self.token
 
     def _refresh_oidc(self, provider):
-        config = Configuration()
+        config = Configuration().get_default_copy()
```

Comment:
Can we unit test this?

Reply:
Who do you mean? As far as I see all tests are passed.
I checked this change locally, and it works.